mne/viz/evoked.py

```diff
@@ -1858,7 +1858,7 @@ def _calculate_ci_and_mean(evokeds, conditions, scaling, picks, ci_fun, gfp):
     return data_dict, ci_dict
 
 
-def _get_ci_function_for_evokeds(ci):
+def _get_ci_function_for_evokeds(ci, do_topo=False):
```

Comment:
@agramfort 

>once you switched to 'auto' option isolate the logic of setting ci with a private function.
Something like _check_ci

like this?

Also see `mne.stats._ci` (ands below 1880)